recipes/docstrfmt/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "docstrfmt" %}
+{% set version = "1.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/docstrfmt-{{ version }}.tar.gz
+  sha256: 9da96e71552937f4b49ae2d6ab1c118ffa8ad6968082e6b8fd978b01d1bc0066
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  entry_points:
+    - docstrfmt = docstrfmt.main:main
+    - docstrfmtd = docstrfmt.server:main
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - flit-core >=3.4,<4
+    - pip
+  run:
+    - python >={{ python_min }}
+    - black ==24.*
+    - click ==8.*
+    - docutils ==0.20.*
+    - libcst ==1.*
+    - platformdirs ==4.*
+    - sphinx >=7,<9
+    - tabulate ==0.9.*
+    - toml ==0.10.*
+    - aiohttp ==3.*
```

Comment:
These super tight dependencies are in the metadata and we can merge this, but this is problematic and prone to error. Only apps should be tightly pinned like this, not libraries.